tests/CatalogTests/CatalogTests.csproj

```diff
@@ -269,9 +269,6 @@
     <PackageReference Include="NuGet.StrongName.json-ld.net">
       <Version>1.0.6</Version>
     </PackageReference>
-    <PackageReference Include="Microsoft.ApplicationInsights">
```

Comment:
Transitive dependency of `NuGet.Services.Logging`. Upgraded to version 2.10.0 as a result.